setup.py

```diff
@@ -10,9 +10,9 @@
 
 
 def _run_build_tables(dir):
-    from subprocess import call
-    call([sys.executable, '_build_tables.py'],
-         cwd=os.path.join(dir, 'pycparser'))
+    from subprocess import check_call
+    check_call([sys.executable, '-B', '_build_tables.py'],
```

Comment:
Would be good to add a comment here explaining why the `-B` and even linking to the PR discussion